include/custom_op/custom_op_lite.h

```diff
@@ -16,19 +16,19 @@ namespace Custom {
 
 class OrtKernelContextStorage : public ITensorStorage {
  public:
-  OrtKernelContextStorage(const OrtW::CustomOpApi& api,
+  OrtKernelContextStorage(const OrtW::CustomOpApi& api_,
```

Comment:
the last _ usually means it's a member of a class, can you change it to something like custom_op_api?

Reply:
sure, will do

Reply:
done